### PR TITLE
Content length

### PR DESCRIPTION
**Pull request by anmol2302**

### CODE REVIEW

File: service/app/util/Filters.java [hunk 1]
```diff
@@ -1,47 +1,42 @@
...
+      Boolean.parseBoolean(ProjectUtil.getConfigValue(JsonKey.SUNBIRD_GZIP_ENABLE));
   private static final String GZIP = "gzip";
   // Size of buffer to use for gzip.
   private static final int BUFFER_SIZE = 8192;
   // Content length threshold, after which the filter will switch to chunking the result.
-  private static final int  CHUNKED_THRESHOLD = 102400;
+  private static final int CHUNKED_THRESHOLD = 102400;
 
-  private Function2<RequestHeader, ResponseHeader, Object> shouldGzip = new AbstractFunction2<RequestHeader, ResponseHeader, Object>() {
+  public static final double sunbird_gzip_size_threshold = 256000.0;
```
Comment (bvinayakumar) [inline service/app/util/Filters.java]: This value should be read from config similar to sunbird_gzip_enable

Comment (bvinayakumar) [inline service/app/util/Filters.java]: Use default value of 262144 (i.e. 256 * 1024)

Comment (anmol2302) [reply service/app/util/Filters.java]: following value set to config file ```export sunbird_gzip_size_threshold=262144```

File: service/app/controllers/BaseController.java [hunk 2]
```diff
@@ -868,4 +876,11 @@ protected String getQueryString(Map<String, String[]> queryStringMap) {
         .map(s -> "?" + s)
         .orElse("");
   }
+
+  public static String getResponseSize(String response) throws UnsupportedEncodingException {
```
Comment (bvinayakumar) [inline service/app/controllers/BaseController.java]: Add unit tests.

File: service/app/util/Filters.java [hunk 3]
```diff
@@ -1,47 +1,41 @@
...
-import org.sunbird.common.models.util.ProjectUtil;
-import org.sunbird.common.models.util.PropertiesCache;
 
 public class Filters implements HttpFilters {
   private EssentialFilter[] filters;
-  private static boolean GzipFilterEnabled = Boolean
-      .parseBoolean(ProjectUtil.getConfigValue(JsonKey.SUNBIRD_GZIP_ENABLE));
+  private static boolean GzipFilterEnabled =
+      Boolean.parseBoolean(ProjectUtil.getConfigValue(JsonKey.SUNBIRD_GZIP_ENABLE));
+  private static final double gZipThreshold =
```
Comment (bvinayakumar) [inline service/app/util/Filters.java]: gzipThreshold

File: service/app/util/Filters.java (hunk 1 above)
Comment (bvinayakumar) [reply service/app/util/Filters.java]: Also, add this in external resource properties file.

**Pull request merged**